python/pyspark/sql/functions/builtin.py

```diff
@@ -14098,10 +14098,13 @@ def element_at(col: "ColumnOrName", extraction: Any) -> Column:
     Notes
     -----
     The position is not zero based, but 1 based index.
+    If extraction is a string, :meth:`element_at` treats it as a literal string,
+    while :meth:`try_element_at` treats it as a column name.
```

Comment:
Is this behavior difference intentional? Is it consistent with the SQL functions element_at and try_element_at?

Reply:
I think the SQL side also treat it as a literal:
```
In [2]: spark.sql("SELECT ELEMENT_AT(MAP('a', 'b'), 'a')").show()
+------------------------+
|element_at(map(a, b), a)|
+------------------------+
|                       b|
+------------------------+
```